osmosis/ADVANCED.md

```diff
@@ -74,25 +74,28 @@ export default () => {
     </div>
   );
 };
+
+export default CounterStore.Provider(Counter);
 ```
 
 ```jsx
 //index.js Root Component
-import React from 'react'
-import { CounterStore } from './counter.store';
+import React from 'react';
+import { StoreProvider } from '@shipt/osmosis';
+import CounterStore from './counter.store';
 import Counter from './counter';
 
 const intervals = [1, 2, 3];
 
-const Counters = () => {
+const App = () => {
   return (
     <>
       {intervals.map(interval => (
         <Counter key={String(interval)} storeKey={interval} interval={interval} />
       ))}
     </>
   );
-});
+};
 
-export default CounterStore.Provider(Counters);
+export default StoreProvider([CounterStore.Provider], App);
```

Comment:
@roni-castro while we're here, mind updating this to `StoreProvider([CounterStore], App);`? Having to include `.Provider` is actually legacy and no longer needed technically

Reply:
@chaceburnette I've updated the code to the new format, and I've also made corresponding updates in this [CodeSandbox](https://codesandbox.io/p/sandbox/osmosis-advanced-fixes-t73tlf?file=%2Fsrc%2FApp.js%3A19%2C51) to validate the changes. Thank you for your feedback!